app/pages/reservation/reservation-confirmation/_reservation-confirmation.scss

```diff
@@ -10,24 +10,31 @@
     text-align: right;
   }
 
-  &__field {
-    padding-top: 12px;
-  }
   a {
     color: $blue;
   }
   &__icon {
     height: 20px;
-    margin-right: 6px;
+    margin-right: 7px;
+  }
+
+  &__highlight {
+    background: darken($light-gray, 5%);
+    padding: 5px 0;
+    margin-bottom: 20px;
   }
 
-  .reservation-date {
+  &__reservation-date {
     .reservation-date__content {
-      margin: 0;
+      margin: 20px auto;
     }
     h3 {
       font-size: 1.6rem;
-      text-align: left;
+      font-weight: normal;
     }
   }
+  &__resource-name {
+    margin-left: 111px;
```

Comment:
These large margin values tend to be problematic on mobile views. Have you tested it?

Reply:
You are right, it looks bad on many different widths. Will differ a bit from layout and center this, it looks OK and avoids any hacks needed to align this text with contents from `ReservationDate`.